frontend/src/components/CommandBar/utils.ts

```diff
@@ -79,32 +85,35 @@ export const useAttributeSearch = (form: FormState<CommandBarSearch>) => {
 	const navigate = useNavigate()
 	const { projectId } = useProjectId()
 	const { commandBarDialog } = useGlobalContext()
+	const { setSearchQuery } = useSearchContext()
+	const { setSearchQuery: setErrorSearchQuery } = useErrorSearchContext()
 	return (
 		attribute: Attribute | undefined,
 		params?: { newTab?: boolean; withDate?: boolean },
 	) => {
 		if (!attribute) return
 
-		const basePath = `/${projectId}/${
-			isErrorAttribute(attribute) ? 'errors' : 'sessions'
-		}`
+		const isError = isErrorAttribute(attribute)
+
+		const basePath = `/${projectId}/${isError ? 'errors' : 'sessions'}`
 		const qbParams = buildQueryBuilderParams({
 			attribute,
 			query,
 			startDate: params?.withDate ? dates[0] : undefined,
 			endDate: params?.withDate ? dates[1] : undefined,
 		})
 
-		const searchQuery = buildQueryURLString(qbParams, {
-			reload: true,
```

Comment:
removed the `?reload` query parameter and corresponding `useEffect`. Setting the search query before navigating has the same effect